scripts/quests/jeuno/LB06_New_Worlds_Await.lua

```diff
@@ -41,10 +41,19 @@ quest.sections =
                 onTrigger = function(player, npc)
                     -- NOTE:
                     -- Event 10045 is the global event used in all Limit break quest from 6 to 10.
-                    -- First argument is mostly 0 in all except here, wich happens to be either player level or level cap.
-                    -- Second or third argument goes up the higher the LB quest
-                    -- The rest, i have no clue. I havent tried it without those, but im inclined to believe they are just unused garbage data.
-                    return quest:progressEvent(10045, 75, 2, 10, 7, 30, 302895, 4095)
+                    -- First argument is player level. (Confirmed from retail captures)
+                    -- Second or third argument are seemingly event control parameters.
+                    -- The rest, I have no clue. I haven''t tried it without those, but im inclined to believe they are just unused garbage data.
+                    return quest:progressEvent(10045, 75, 2, 2964, -1, -1075052545, 2147434431, 3)
```

Comment:
I'm fairly certain that only the first two parameters are relevant.  First param always seems to be the player's current level (which may have no impact with our implementation), and the second seems to be the real driver.

For this and the below changes, some experimentation should be done to see what impacts actual change, and what does not.

Reply:
Since there was an event update involved, I added those just purely for documentation